docs/user/filter-and-process/custom-pipeline-processing-ottl.md

```diff
@@ -0,0 +1,161 @@
+# Custom Pipeline Processing with OTTL
+
+> **Note**: This feature is only available with Telemetry Manager v1.52.0 and later.
+
+After your telemetry data has been collected, you can use the custom transform and filter feature with the [OpenTelemetry Transformation Language (OTTL)](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/pkg/ottl) to apply advanced, fine-grained processing to your telemetry data, enriching, modifying, and filtering it before it is exported to a backend.
+
+You can define `transform` and `filter` OTTL rules in the `spec` of your LogPipeline, MetricPipeline, and TracePipeline resources. When you define rules, the Telemetry module configures processors in the underlying OpenTelemetry Collector to execute your statements.
+
+## Processing Order
+
+Processing rules are always applied in a specific order. This is critical to understand when designing your rules:
+
+1. **Transformations (transform)**: Data is first modified according to the rules in the `transform` section
+2. **Filters (filter)**: The transformed data is then evaluated against the rules in the `filter` section. Any data matching a filter condition is dropped
+
+This sequence ensures that you filter on the final, transformed state of your data. For example, if you rename an attribute in a transform rule, your filter rule must use the new name, not the original one.
+
+## OTTL Context and Data Processing
+
+Transform and filter operations work at different context levels depending on the telemetry data type. Currently, custom context configuration is not supported, and all operations use the lowest-level context by default:
+
+| Pipeline Type  | Default Context | Description                                       | Requirements              |
```

Comment:
Is it really important to say that the lowest-level context is used? Is it even true for transformations? Isn't it more important to say that we support only the inferred context approach and that we do not support spanevents?

So at the end removing this block here and start with the next block and put the Note afterwards?